pkg/vtpm/src/server.cpp

```diff
@@ -23,13 +26,18 @@ using namespace google::protobuf::io;
 #define CODED_STRM_HDR_LEN 4
 #define SERVER_PORT 8877
 #define LISTEN_BACKLOG_LIMIT 16
-#define MAX_ARGS 15
+#define MAX_ARGS 25
 #define BIN_PATH "/usr/bin/"
-#define CMD_OUTPUT_FILE "cmd.output"
+#define CLIENT_READ_TIMEOUT 2  //seconds
```

Comment:
Given that we have a thread per domU/client we can make this value much larger. I'd say a minute is reasonable so that things don't fail of a client is heavily loaded.

Reply:
Changed it to 60 sec